crates/astria-tracing-tower/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+/// A vendored copy of the unpublished [`tracing-tower`] crate:
+///
+/// Based on [`penumbra-tracing-tower`].
+/// [`tracing-tower`]: https://github.com/tokio-rs/tracing/tree/e603c2a254d157a25a7a1fbfd4da46ad7e05f555/tracing-towerpub mod trace;
```

Comment:
```suggestion
/// [`tracing-tower`]: https://github.com/tokio-rs/tracing/tree/e603c2a254d157a25a7a1fbfd4da46ad7e05f555/tracing-tower
```

I think?

Reply:
heh, it complained about `trace` not being available, so that's where it went. Either way, removed the dep in favour of just using penumbra.